src/map/pc.cpp

```diff
@@ -12006,38 +12120,35 @@ static bool pc_readdb_skilltree(char* fields[], int columns, int current)
 
 /** [Cydh]
 * Calculates base hp of player. Reference: http://irowiki.org/wiki/Max_HP
-* @param level Base level of player
-* @param class_ Job ID @see enum e_job
+* @param level: Base level of player
+* @param job_id: Job ID @see enum e_job
 * @return base_hp
 */
-static unsigned int pc_calc_basehp(uint16 level, uint16 class_) {
-	double base_hp;
-	uint16 i, idx = pc_class2idx(class_);
+static unsigned int pc_calc_basehp(uint16 level, uint16 job_id) {
```

Comment:
Both functions should only be available if the HP_SP_TABLES is not defined.
Code and warnings/errors should be changed accordingly.